src/tokens/borderRadius.js

```diff
@@ -7,7 +7,7 @@ const light = core => ({
   // Checkbox
   Checkbox: core.BorderRadius.Small,
   // Chip
-  Chip: core.BorderRadius.Small,
+  Chip: 12,
```

Comment:
Does it not work to use the XLarge variant?

Reply:
Wait, I just tried it, it doesnt change the look at all 😅

Reply:
Well, if you want to change it to reference `core.BorderRadius.XLarge` we can push up the change.